server/src/services/users.ts

```diff
@@ -6,6 +6,13 @@ async function getUserByAuth0Id(
   return await UserModel.findOne({ auth0_id: auth0Id });
 }
 
+async function create(user: UserAttributes): Promise<UserAttributes> {
+  return UserModel.create({
+    ...user,
+    created_at: new Date(),
+  });
+}
+
 const UserService = {
   getUserByAuth0Id,
 };
```

Comment:
We're gonna be using UserService.create in another file.
How are we supposed to use it